src/systems/detachable_joint/DetachableJoint.hh

```diff
@@ -73,6 +86,15 @@ namespace systems
                 const ignition::gazebo::UpdateInfo &_info,
                 ignition::gazebo::EntityComponentManager &_ecm) final;
 
+    /// \brief Gazebo communication node.
+    private: transport::Node node;
+
+    /// \brief A publisher to send state of the detachment
+    private: ignition::transport::Node::Publisher outputPub;
```

Comment:
nit: You can drop the `ignition` to make it easier when it's forward ported to the `gz` namespace.

Reply:
addressed [here](https://github.com/gazebosim/gz-sim/pull/1687/commits/4154296935bea0590eba0e61795cd0d0372e8e69#diff-087c0092b0bf11de46dfb2b8ce617f244227ec4eb93b1adfbc04229b9f0457c7R93)